### ISSUE TITLE
Use of deprecated function causes error

### ISSUE THREAD

**Issue by jboulter11** [kickingvegas/YmsCoreBluetooth#110]
<img width="234" alt="screenshot 2015-09-21 18 58 39" src="https://cloud.githubusercontent.com/assets/6633472/10007403/e2a9344c-6092-11e5-9173-1d6acfc40cce.png">

YMSCBPeripheral currently uses a deprecated method, [CBPeripheralDelegate peripheralDidInvalidateServices], which has now been removed from Core Bluetooth.  This issue prevents compilation and use of the framework.

Is there a proposed solution?

Thank you!

**Comment by jboulter11** [comment]
<img width="226" alt="screenshot 2015-09-21 19 03 55" src="https://cloud.githubusercontent.com/assets/6633472/10007459/7f2c5308-6093-11e5-8467-d7eb27417bd5.png">

It appears there are other deprecated methods as well.

**Comment by 501336North** [comment]
yup, I was using this pod and liked it a lot.  But looks like it's no longer compatible with iOS9  it just won't compile anymore.  As jboulter11 stated there are a few problems.  Looking for a fix and/or to know if that repo is maintained.

TIA

**Comment by kickingvegas** [comment]
First off, thanks for your interest in YmsCoreBluetooth! Apologies for not getting around to updating this library but I'll be working on updating it for iOS 9 this weekend and should have it available by Monday.

**Comment by jboulter11** [comment]
That's fantastic news!  Thank you for your reply!

**Comment by 501336North** [comment]
Awesome, thanks a bunch for addressing the issue in a timely fashion... this is just fabulous.  You make my day!

**Comment by kickingvegas** [comment]
Fix merged in with #112. Look for CocoaPods update soon.

**Comment by JohannesRudolph** [comment]
Can be closed in my opinion, 1.10 is working fine for me.
